src/elem.js

```diff
@@ -265,6 +265,10 @@ exports.renderToStaticHtml = function(el) {
 
 exports.el = el;
 
+exports.jsx = function(name, attrs, children) {
```

Comment:
I don't think it's going to work because JSX doesn't create an array of children but use `arguments` instead. You need to spread the arguments with something like :+1: 

```javascript
export.jsx = function(type, attributes) {
  for (var _len3 = arguments.length, children = Array(_len3 > 2 ? _len3 - 2 : 0), _key3 = 2; _key3 < _len3; _key3++) {
    children[_key3 - 2] = arguments[_key3];
  }
  return el(type, attributes || {}, children);
}
```